docs/javascript/loadsamples.js

```diff
@@ -1,7 +1,12 @@
 /**
  * This file is unique for each sample browser. It contains the logic specific to each repo for loading the samples as needed.
  */
-var jsonPath = "https://raw.githubusercontent.com/pnp/teams-dev-samples/gh-pages/samples.json";
+var jsonPath = "https://pnp.github.io/teams-dev-samples/samples.json";
+if (window.location.host.toLowerCase() !== "pnp.github.io") {
```

Comment:
Technically, we could always load from the window.location.host URL (without the `if` statement), but this works great!

Reply:
Thanks; I think we discussed that one offline and it is resolved; thanks!